awx/main/migrations/0084_v360_add_notificationtemplate_messages.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
```

Comment:
@jladdjr you and @beeankha are gonna compete for ownership of this one (whoever merges first), so keep an eye on it.

Reply:
👍